### ISSUE TITLE
(Un)focus without closing using keyboard

### ISSUE THREAD

**Issue by jasonpincin** [platformio/platformio-atom-ide-terminal#23]
Would be nice to be able to leave the terminal open and refocus editing panes from terminal or terminal from editing panes, all with the keyboard. 
##

**Comment by ChillySpark** [comment]
Yes, please consider adding this feature. As a workaround, I've been doing both of these:
1. type ctrl-` to toggle terminal open/close (focus goes back & forth)
2. type ctrl-00 from the open terminal to bring focus back to editing pane (only one way)
